package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "dbstream-fs",
-  "version": "1.0.7",
+  "version": "1.1.0",
```

Comment:
Should we regenerate the `package-lock.json` file to have the updated version?

<img width="438" alt="Screen Shot 2022-08-31 at 6 58 59 PM" src="https://user-images.githubusercontent.com/43310906/187724694-6a0e2756-a00a-4245-863d-1985ed5e2cbb.png">